creating_images/guidelines.adoc

```diff
@@ -265,6 +265,14 @@ In order to support running containers with volumes mounted in a secure fashion,
 
 Lastly, the final *_USER_* declaration in the Dockerfile should specify the user id (numeric value) and not the username.  This allows OpenShift to validate the authority the image is attempting to run with and prevent running images that are trying to run as root because running containers as a privileged user exposes potential security holes.  If the image does not specify a *_USER_* it inherits the *_USER_* from the parent image.
 
+[IMPORTANT]
+====
+If your S2I image does not include a USER declaration with a numeric user, your builds will fail by default. In order to
+allow images that use either named users or the root (0) to user to build in OpenShift, you must add the project's builder
+service account (`system:serviceaccount:[your-project]:builder`) to the privileged Security Context Constraint, following
+link:../admin_guide/manage_scc.html#grant-access-to-the-privileged-scc[these instructions].
```

Comment:
or change the restricted SCC as described here, right?
https://docs.openshift.org/latest/admin_guide/manage_scc.html#enable-images-to-run-with-user-in-the-dockerfile